command/cli.go

```diff
@@ -129,14 +129,16 @@ type FixArgs struct {
 
 func (va *ValidateArgs) AddFlagSets(flags *flag.FlagSet) {
 	flags.BoolVar(&va.SyntaxOnly, "syntax-only", false, "check syntax only")
+	flags.BoolVar(&va.EvaluateDatasources, "evaluate-datasources", false, "evaluate datasources for validation (may incur costs)")
```

Comment:
I'm not sure on the copy but should we indicate that this is for HCL2 templates only?

Reply:
I can add this in the message sure, doesn't hurt to be precise